asynch/connection.py

```diff
@@ -220,52 +196,33 @@ async def ping(self) -> None:
             msg = f"Ping has failed for {self}"
             raise ConnectionError(msg)
 
+    async def _refresh(self) -> None:
+        """Refresh the connection.
 
-async def connect(
-    dsn: Optional[str] = None,
-    user: str = constants.DEFAULT_USER,
-    password: str = constants.DEFAULT_PASSWORD,
-    host: str = constants.DEFAULT_HOST,
-    port: int = constants.DEFAULT_PORT,
-    database: str = constants.DEFAULT_DATABASE,
-    cursor_cls=Cursor,
-    echo: bool = False,
-    **kwargs,
-) -> Connection:
-    """Return an opened connection to a ClickHouse server.
-
-    Equivalent to the following steps:
-    1. conn = Connection(...)  # init a Connection instance
-    2. conn.connect()  # connect to a ClickHouse server
-    3. return conn
-
-    When the connection is no longer needed,
-    consider `await`ing the `conn.close()` method.
-
-    :param dsn str: DSN/connection string (if None -> constructed from default dsn parts)
-    :param user str: user string ("default" by default)
-    :param password str: password string ("" by default)
-    :param host str: host string ("127.0.0.1" by default)
-    :param port int: port integer (9000 by default)
-    :param database str: database string ("default" by default)
-    :param cursor_cls Cursor: Cursor class (asynch.Cursor by default)
-    :param echo bool: connection echo mode (False by default)
-    :param kwargs dict: connection settings
-
-    :return: an opened Connection object
-    :rtype: Connection
-    """
-
-    conn = Connection(
-        dsn=dsn,
-        user=user,
-        password=password,
-        host=host,
-        port=port,
-        database=database,
-        cursor_cls=cursor_cls,
-        echo=echo,
-        **kwargs,
-    )
```

Comment:
This function is unnecessary. A connection is instantiated and returned, but the user should not forget to close it manually. So it could be decorated with an `asynccontextmanager` with resource clean-up under the hood. But its corresponding class (here `Connection`) can be used in an `async with` context, so no need to duplicate this feature. That is why this function is removed and moving to the version 0.3.0 is a legitimate explanation for getting rid of this redundant routine.